py_ocsf_models/objects/metadata.py

```diff
@@ -83,7 +83,7 @@ class Metadata(BaseModel):
     correlation_uid: Optional[str]
     event_code: Optional[str]
     uid: Optional[str]
-    labels: Optional[List[str]]
+    labels: Optional[dict]
```

Comment:
This will not comply with the OCSF specification https://schema.ocsf.io/1.2.0/objects/metadata?extensions=. `labels` must be a String Array.